### ISSUE TITLE
Strange Error on Package Installation

### ISSUE THREAD

**Issue by jonrobinson2** [RevolutionAnalytics/RRO#92]
I get these errors when I try to install more recent packages using RRO. Is this because they aren't on the MRAN or there something else going on here? Both packages are available for R3.1.1

caretEnsemble: Depends:	R (≥ 3.1.0)
rvest: Depends:	R (≥ 3.0.1)

> install.packages("caretEnsemble")
Installing package into ‘/home/jrobinson/R/x86_64-unknown-linux-gnu-library/3.1’
(as ‘lib’ is unspecified)
Warning message:
package ‘caretEnsemble’ is not available (for R version 3.1.1)
> install.packages("rvest")
Installing package into ‘/home/jrobinson/R/x86_64-unknown-linux-gnu-library/3.1’
(as ‘lib’ is unspecified)
Warning message:
package ‘rvest’ is not available (for R version 3.1.1)

> sessionInfo()
R version 3.1.1 (2014-07-10)
Platform: x86_64-unknown-linux-gnu (64-bit)

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C
 [9] LC_ADDRESS=C               LC_TELEPHONE=C
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base

other attached packages:
[1] colorout_1.0-3

loaded via a namespace (and not attached):
[1] tools_3.1.1

**Comment by andrie** [comment]
This is most likely because you are using the default MRAN snapshot.

See http://mran.revolutionanalytics.com/open/#repos

For RRO-8.0.1, which is based on R-3.1.2, the default MRAN snapshot date is "2014-12-01"

To change your repos date to a different date, follow the instructions at http://mran.revolutionanalytics.com/faq/#change-repos

**Comment by jonrobinson2** [comment]
Thanks @andrie I'm an idiot for not reading far enough into the docs!

**Comment by revodavid** [comment]
Specifically, you will need a repository after Jan 16 2015, which is when caretEnsemble was released. This is easy with `checkpoint`:

```
> checkpoint("2015-01-25")
...
> install.packages("caretEnsemble")
```

Also it looks like you are using RRO 8.0.0, based on R 3.1.1. You might want to upgrade to RRO 8.0.1 (based on R 3.1.2).

**Comment by jonrobinson2** [comment]
Any specific reason to upgrade to the sub point 1 release?

**Comment by revodavid** [comment]
It's based on R 3.1.2, which is the latest R engine. caretEnsemble was released after R 3.1.2 was, so you may avoid some compatibility issues with it or one of its many dependencies.

**Comment by jonrobinson2** [comment]
Thanks. I figured that was the case, was more curious if the sub point 1 release for RRO was any different from the sub point 0 release besides the different R engine.
